conf/janus.transport.rabbitmq.jcfg.sample

```diff
@@ -24,10 +24,21 @@ general: {
 	#username = "guest"					# Username to use to authenticate, if needed
 	#password = "guest"					# Password to use to authenticate, if needed
 	#vhost = "/"						# Virtual host to specify when logging in, if needed
-	to_janus = "to-janus"				# Name of the queue for incoming messages
-	from_janus = "from-janus"			# Name of the queue for outgoing messages
+
 	#janus_exchange = "janus-exchange"	# Exchange for outgoing messages, using default if not provided
 	#janus_exchange_type = "fanout" 		# Rabbitmq exchange_type can be one of the available types: direct, topic, headers and fanout (fanout by defualt).
+
+	#queue_name = "janus-gateway" # Queue name for incoming messages (if this is set and janus_exchange_type is topic, to_janus will be the topic the queue is bound to the exchange on)
+
+	to_janus = "to-janus"				# Name of the queue for incoming messages if queue_name isn't set, otherwise, the topic that queue_name is bound to
+	from_janus = "from-janus"			# Topic of the message sent from janus
+
+	#declare_outgoing_queue = true # By default (for backwards compatibility), we declare an outgoing queue. Set this to false to disable that behavior
```

Comment:
clarify that `from_janus` will be the name of the outgoing queue
same for admin